packages/amplify-graphql-generation-transformer/src/utils/tools.ts

```diff
@@ -0,0 +1,73 @@
+import { TransformerContextProvider } from '@aws-amplify/graphql-transformer-interfaces';
+import { FieldDefinitionNode } from 'graphql';
+import { JSONSchema, generateJSONSchemaFromTypeNode } from './graphql-json-schema-type';
+
+export type Tool = {
+  toolSpec: ToolSpec;
+};
+
+export type Tools = {
```

Comment:
Do you use this type elsewhere?

Reply:
We don't, and your comment made me realize that we only actually need to export `ToolConfig`. The extra `export`s and `Tools` type definition are relics from previous structure. Will remove in a follow up. Thanks for the callout!